spyder/plugins/explorer/widgets.py

```diff
@@ -129,7 +129,7 @@ def icon(self, icontype_or_qfileinfo):
         else:
             qfileinfo = icontype_or_qfileinfo
             fname = osp.normpath(to_text_string(qfileinfo.absoluteFilePath()))
-            return ima.get_icon_by_extension(fname)
+            return ima.get_icon_by_extension(fname, 1.0)

```

Comment:
Please add here `scale_factor` to the keyword args, like this:

    return ima.get_icon_by_extension(fname, scale_factor=1.0)